sources/packages/forms/src/form-definitions/institutionprofile.json

```diff
@@ -23,7 +25,27 @@
         "json": ""
       },
       "customConditional": "",
-      "logic": [],
+      "logic": [
+        {
+          "name": "Client type Not AEST",

```

Comment:
for the sake of consistency either `Client Type Not AEST` or `Client type not AEST`